e2e/e2etests/test_eth_deposit.go

```diff
@@ -20,16 +20,29 @@ import (
 )
 
 // TestEtherDeposit tests deposit of ethers
-func TestEtherDeposit(r *runner.E2ERunner) {
-	hash := r.DepositEtherWithAmount(false, big.NewInt(10000000000000000)) // in wei (0.01 eth)
+func TestEtherDeposit(r *runner.E2ERunner, args []string) {
+	defaultAmount := big.NewInt(10000000000000000) // default value of 0.01 ETH in wei
+	amount := new(big.Int).Set(defaultAmount)
+
+	if len(args) > 1 {
+		r.Logger.Error("TestEtherDeposit accepts only one argument for the amount, using default value")
```

Comment:
I think we can instead throw an error in the test if argument not valid (which is currently done by panicking)